staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder/builder_test.go

```diff
@@ -592,7 +592,7 @@ func TestBuildSwagger(t *testing.T) {
 					Scope:      apiextensions.NamespaceScoped,
 					Validation: validation,
 				},
-			}, "v1")
+			}, "v1", Options{V2: true, StripDefaults: true})
```

Comment:
do you think it would be worth extending this test and exercise `StripValueValidations` and `StripDefaults` methods?

Reply:
yes, we should add some smoke tests.

Reply:
done